statistics-production/ud.qmd

```diff
@@ -817,6 +817,77 @@ for dashboards and other secondary statistics services, whilst also helping with
 
 ---
 
+## Sex and gender
+
+The Department's policy on collecting sex and gender data is that statistics 
+should preferentially be presented for sex rather than gender. These terms, as 
+used in statistics publications, are defined in the following sections and 
+publication teams should be careful not to mislabel sex categories as gender or
+vice-versa.
+
+---
+
+### Sex
+
+---
+
+Sex as reported by DfE statistics, is:
+
+> This is a value which identifies the sex of a person as recognised in law, 
+i.e. the sex as recorded on a birth certificate (or on a gender recognition 
+certificate).
+
+When presented in a filter, this should take the column header **sex** and 
+standard entries **Male**, **Female** and **Unknown**. This can also be included as 
+part of the standard **breakdown_topic** / **breakdown** filters, with the relevant 
+entry under **breakdown_topic** being **Sex** and as above, the entries under
+**breakdown** being **Male**, **Female** and **Unknown**.
```

Comment:
Would it be possible to give a small table of the allowable values as well (at the bottom of the sex section) so it's visual and similar to how analysts would see it in a CSV?

Reply:
Added that now